README.md

```diff
@@ -3,7 +3,7 @@
 ![Build and test](https://github.com/cqfn/diKTat/workflows/Build%20and%20test/badge.svg)
```

Comment:
I guess these urls need to be updated too:
`https://github.com/cqfn/diKTat` to `https://github.com/saveourtool/diktat`

Reply:
@nulls, this has been addressed with 49bbef290f19f352c46ba06eb75506cb1957ada5.